LICENSE.md

```diff
@@ -1,10 +1,10 @@
-## This project is in the worldwide public domain
+# License
 
-As a work of the United States government, this project is in the public domain within the United States.
+## This project is in the worldwide public domain
 
-Additionally, we waive copyright and related rights in the work worldwide through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+As a work of the United States government, this project is in the public domain within the United States under the [National Institute of Standards and Technology License](https://www.nist.gov/open/license) (copied below).
 
-As a work of the United States government, this project is in the public domain within the United States under the [National Institute of Standards and Technology License](https://www.nist.gov/open/license)). Additionally, copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+Additionally, copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Do we imply through these 2 paragraphs that we have a bi-standard license: one for the US and another one worldwide? We should settle on only one License.

Reply:
@aj-stein-nist has made the same point. I was attempting to update, while keeping 'the good parts' from the Metaschema repo. Now planning to roll the LICENSE  back to the normative/standard/simple one.